examples/exec.py

```diff
@@ -2,7 +2,7 @@
 
 from kubernetes import config
 from kubernetes.client import Configuration
-from kubernetes.client.apis import core_v1_api
+from kubernetes.client.api import core_v1_api
```

Comment:
Is this right? I get an error: `ModuleNotFoundError: No module named 'kubernetes.client.api'`.

Reply:
could you print the package info in your code, to see if it's using the updated code from this pull? I suspect you have official kubernetes package installed in your env and the code is using  that by default

Reply:
You can also try modifying your PYTHONPATH variable if you’re under Linux and prioritize searching for the k8s package locally.